Fedora/input/xccdf/system/accounts/physical.xml

```diff
@@ -122,6 +122,51 @@ the grub2 superuser account and password, please refer to
 <ref nist="IA-2(1),IA-5(e),AC-3" disa="213" cis="1.5.3" />
 </Rule>
 
+<Rule id="bootloader_uefi_password" severity="medium">
+<title>Set the UEFI Boot Loader Password</title>
+<description>The UEFI grub2 boot loader should have a superuser account and password
+protection enabled to protect boot-time settings.
+<br /><br />
+To do so, select a superuser account and password and add them into the
+appropriate grub2 configuration file(s) under <tt>/etc/grub.d</tt>.
```

Comment:
This wiki page clarifies it's actually the very same file name for Fedora system:
  https://fedoraproject.org/wiki/GRUB_2?rd=Grub2

Please scroll down to ```Setting a password for interactive edit mode``` section.